### PR TITLE
#317 Provide classname MBeanInfo on List request

### PR DESCRIPTION
**Pull request by apupier**
Signed-off-by: Aurelien Pupier <apupier@redhat.com>

### CODE REVIEW

File: agent/core/src/test/java/org/jolokia/handler/ListHandlerTest.java [hunk 1]
```diff
@@ -168,7 +168,7 @@ public void truncatedList() throws Exception {
         JmxListRequest request = new JmxRequestBuilder(RequestType.LIST).pathParts("java.lang", "type=Runtime").build();
         Map res = execute(request);
         assertFalse(res.containsKey("op"));
-        assertEquals(res.size(),2);
+        assertEquals(res.size(),3);
```
Comment (apupier) [inline agent/core/src/test/java/org/jolokia/handler/ListHandlerTest.java]: @rhuss for this one I'm not completely sure if it is coherent

Comment (apupier) [inline agent/core/src/test/java/org/jolokia/handler/ListHandlerTest.java]: and might need to update https://github.com/rhuss/jolokia/blob/b6010e3c053648eb280e57c1255f741d579b0688/src/docbkx/protocol/list.xml#L158 then, will do it tomorrow if your answer is positive

Comment (rhuss) [inline agent/core/src/test/java/org/jolokia/handler/ListHandlerTest.java]: Should be fine. The only key missing here is "op" as the RuntimeMBean does not have an operations.

Would be cool if you could update the docs, too (and maybe replace "Reduce the depth of depth" by "Reduce the depth of the tree" while you are at it ;-)

Comment (apupier) [inline agent/core/src/test/java/org/jolokia/handler/ListHandlerTest.java]: done

**Comment by rhuss** [comment]
Thanks, looks good !

**Pull request merged**